content/news/releases/prestashop-8-0-1-maintenance-release.md

```diff
@@ -4,14 +4,14 @@ title:  PrestaShop 8.0.1 is available
 subtitle: Maintenance version of the 8.0.x branch
 date: 2023-01-06
 authors: [ PrestaShop ]
-image: /assets/images/2023/01/release801.png
+opengraph_image: /assets/images/2023/01/release801.png
 twitter_image: /assets/images/2023/01/release801.png
```

Comment:
If it's the same image, why not set up just one?

Reply:
Finally, it won't be the same image because the dimensions will differ. I left it there to see if it works after the deploy.